CHANGELOG.md

```diff
@@ -1,13 +1,24 @@
 # Change Log - @itwin/insights-client
 
+## 0.4.0
+
+Thu, 3 Mar 2023
+
+### Minor
+
+- Removed `url` dependency.
+- Refactored OCLCA.
```

Comment:
This is not just a refactoring. This "refactoring" contains breaking changes. Please write what changed so that user knows what to update in their application (e.g. no longer using `IMJS_URL_PREFIX` env variable).

Reply:
Sorry, was in the middle of writing it. Finished it. Let me know if its unclear.